versions/development/SPEC.md

```diff
@@ -358,6 +358,96 @@ Strings can also contain the following types of escape sequences:
 * A hexadecimal escape code starts with `\x`, followed by 2 hexadecimal digits `0-9a-fA-F`. 
 * A unicode code point starts with `\u` followed by 4 hexadecimal characters or `\U` followed by 8 hexadecimal characters `0-9a-fA-F`.
 
+##### Multi-line Strings
+
+Strings that begin with `<<<` and end with `>>>` may span multiple lines.
+
+```wdl
+String s = <<<
+  This is a
+  multi-line string!
+>>>
+
+In multi-line strings, leading *whitespace* is removed according to the following rules. In the context of multi-line strings, whitespace refers to space (`\x20`) and tab characters only and is treated differently from newline characters.
+
+1. Remove all whitespace following the opening `<<<`, up to and including a newline (if any).
+2. Remove all whitespace preceeding the closing `>>>`, up to and including a newline (if any).
+3 Remove all line continuations and subsequent white space.
+  * A line continuation (`\`) comes at the end of a line (i.e. immediately preceding a newline) and indicates that two consecutive lines are actually the same line (e.g. when breaking a long line for better readability).
+  * Removing a line continuation means removing the `\` character, the immediately following newline, and all the whitespace preceeding the next non-whitespace character or end of line (whichever comes first).
```

Comment:
Can I suggest processing the line continuations before the opening/closing whitespace? This avoids having to clarify+implement a couple of tedious corner cases, where the first or last newline is an escaped one.

Missing period after 3

Reply:
Another nit is that the backslash at the end of a line could itself be escaped, 

```
<<<abc\\
def>>> 
=> "abc\\\ndef"
```

I think we can say that a continued line ends in an *odd number* of backslashes.

Reply:
Thanks, I made both those changes.